Singular.js

```diff
@@ -1,11 +1,15 @@
 import {NativeEventEmitter, NativeModules} from 'react-native';
 import {version} from './package.json';
+import { Platform, StyleSheet } from 'react-native';
```

Comment:
do we use StyleSheet?
And btw, we already import from react native on line 1 so we can just append the imports there